data/json/items/melee/bludgeons.json

```diff
@@ -1137,5 +1137,23 @@
     "cutting": 23,
     "price": 16000,
     "qualities": [ [ "HAMMER", 1 ] ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "battery_sock",
+    "symbol": ";",
+    "color": "white",
+    "looks_like": "rock_sock",
+    "name": { "str": "sock full o' batteries", "str_pl": "a sock full of old batteries" },
+    "description": "A pair of socks full of old disposable batteries--the poor man's nunchaku.",
```

Comment:
```suggestion
    "description": "A pair of socks full of disposable batteries--the poor man's nunchaku.",
```

Reply:
A fair point. It's not like they get heavier with age.